src/book.py

```diff
@@ -129,12 +131,14 @@ def clean(self, lemmatize=True, deromanize=False, lemma_pos='v', inplace=False):
         inplace (default False): replaces the text of the object with cleaned text
         """
         cleaned = self._text
+        '
 
-        garbage = '\ufeff|â€™|â€"|â€œ|â€˜|â€\x9d|â€œi|_|â€'
+        garbage = '\ufeff|â€™|â€"|â€œ|â€˜|â€\x9d|â€œi|_|â€|£|éé|ô|à|â|ê|—£|éè|ü|é|œ|î|æ|ç|‘|é—|…|ö|è'
```

Comment:
Are all of these garbage characters that should be removed? Or should the non-accented vowels be replaced with accented vowels, and the ellipses (...) be replaced with a space? I don't know where in the texts you've come across these, but we should probably try to infer what the letters actually are.

In modern English, oe and ae are usually replaced with just "e"

Reply:
Most of them are garbage but some can be subbed for if we need more words in our vocabulary.
Lets put that on the TODO list for now. In case we decide to make the correct substitutions we can use this list [https://en.wikipedia.org/wiki/List_of_words_that_may_be_spelled_with_a_ligature]